### ISSUE TITLE
Upload missing Videos & Photos if WLAN near.

### ISSUE THREAD

**Issue by xadox-1st** [nextcloud/android#549]
### Actual behaviour
-Actually uploading is only enabled while in WLAN for me. Pictures taking without WLAN will never be uploaded.

### Expected behaviour
-Some kind of sync. So all not already uploaded photos will be uploaded when again in WLAN.
 

### Environment data
Android version: 6.0 EMUI 4.1.1
Device model: Huawei P9 Lite
Nextcloud app version: 1.4.0
Nextcloud server version: 10.?

**Comment by heyiswas** [comment]
Same here (same device and version). Nextcloud Server is v11.

**Comment by tobiasKaminsky** [comment]
@mario as you are now also familiar with upload.
Do you have an idea? This seems to be instantUpload as it is Android 6...?

**Comment by tobiasKaminsky** [comment]
@xadox-1st @heyiswas do you see the upload as pending in the upload view?

**Comment by heyiswas** [comment]
No pending upload. It's like nothing happens at all. Also it makes no difference, if the Nextcloud app is running or not.

**Comment by mario** [comment]
@tobiasKaminsky yes, this is instant upload - folks, will you be able to test our new beta build tomorrow? We enabled an improved AutoUpload for Android 6 as well, so that should help with this problem.

You should be able to download it from this link (but tomorrow please) - https://github.com/nextcloud/android/raw/beta/apks/latest.apk

(Let me know if I'm wrong on the link @tobiasKaminsky)

**Comment by heyiswas** [comment]
@mario thanks, I'll test it.

**Comment by tobiasKaminsky** [comment]
As an a alternative you can get it also from f-droid, but as they have an one day delay only try it tomorrow.

**Comment by heyiswas** [comment]
The auto upload with the beta app (over radio) works fine. Thanks for your work!

**Comment by mario** [comment]
@heyiswas aware of the issue. Working on it ;)

**Comment by mario** [comment]
Can you please try the newest beta?

https://github.com/nextcloud/android/raw/beta/apks/latest.apk

**Comment by heyiswas** [comment]
@mario I'll check it out later. :)

**Comment by mario** [comment]
Thanks! :)

**Comment by heyiswas** [comment]
Hey @mario everything seems to be fine now. I've tested the following scenarios:

1. Configured one local folder for the auto upload over WIFI, while I was not connected to my WIFI. After adding one new picture to my local folder and connecting to my WIFI, there was one pending upload, but nothing happend. Then I just configured a second folder for auto upload over WIFI, added a file to this folder and BAM, the changes of both folders got uploaded! :100: I can't tell you, why.

2. Configured my camera roll for the auto upload over WIFI. Snapped a picture and it got recognized as an pending upload, but didn't start. After ~20 minutes, the upload started and finished perfectly!

Good job, mario! :+1:
